src/_tutorials/server/httpserver.md

```diff
@@ -856,22 +856,18 @@ also uses the http_server package.
 This server serves any file from the server's directory
 or subdirectory.
 
-Run `static_file_server.dart`,
-and test it with the URL localhost:4048/file.txt.
-Change `file.txt` to other filenames within the directory.
+Run `static_file_server.dart`, and test it with the URL
+[localhost:4048](http://localhost:4048).
```

Comment:
@johnpryan Do you think it's OK to have links to localhost? (I think we've been inconsistent, and I don't know what the considerations are.)

Reply:
FYI, we have links to localhost on all three sites (Dart, Flutter and NgDart). But of course, I'm open to any other considerations that there might be.

Reply:
It looks like the link checker is working, so from a tooling perspective this seems fine. AFAICT this works on all platforms, and isn't a security issue.